pkg/instrumentation/sdk.go

```diff
@@ -410,7 +413,10 @@ func (i *sdkInjector) injectCommonSDKConfig(ctx context.Context, otelinst v1alph
 	return pod
 }
 
-func chooseServiceName(pod corev1.Pod, resources map[string]string, index int) string {
+func chooseServiceName(pod corev1.Pod, useLabelsForResourceAttributes bool, resources map[string]string, index int) string {
```

Comment:
can we add a docstring for this method that includes the precedence?

Reply:
also added a docstring to similar methods